astropy/tests/helper.py

```diff
@@ -604,3 +604,40 @@ def check_pickling_recovery(original, protocol):
     class_history = [original.__class__]
     generic_recursive_equality_test(original, unpickled,
                                     class_history)
+
+
+def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=0, err_msg='', verbose=True):
+    """
+    Raise an assertion if two objects are not equal up to desired tolerance.
```

Comment:
Actually it doesn't matter, right? ``atol`` has to be specified as a quantity to work so the units have to be explicit anyway. The only reason it doesn't have units by default is because for zero it's unambiguous and doesn't matter what the units are. But you can do:

```
assert_quantity_allclose([1,2] * u.m, [100,200] * u.cm, atol=2 * u.cm)
```

and that should work.